MHTimer.js

```diff
@@ -139,6 +146,61 @@ function messageParse(message) {
             }
             // console.log(typeof retStr);
             break;
+        case 'remind':
+            if (tokens.length === 1) {
```

Comment:
i would rename tokens to user_message_parts or something more descriptive in the future